pkg/zfs/mount.go

```diff
@@ -208,11 +208,37 @@ func MountDataset(vol *apis.ZFSVolume, mount *apis.MountInfo) error {
 }
 
 // MountVolume mounts the disk to the specified path
-func MountVolume(vol *apis.ZFSVolume, mount *apis.MountInfo) error {
+func MountFilesystem(vol *apis.ZFSVolume, mount *apis.MountInfo) error {
 	switch vol.Spec.VolumeType {
 	case VOLTYPE_DATASET:
 		return MountDataset(vol, mount)
 	default:
 		return MountZvol(vol, mount)
 	}
 }
+
+func MountBlock(vol *apis.ZFSVolume, mountinfo *apis.MountInfo) error {
+	target := mountinfo.MountPath
+	devicePath := ZFS_DEVPATH + vol.Spec.PoolName + "/" + vol.Name
+	mountopt := []string{"bind"}
```

Comment:
Just noticed this. Not for this PR, but can you check what is the process of capturing the mountOptions specified via StorageClass and using them. Does MountInfo come with those parameters?

Reply:
mount option has no meaning for block volume. Even if storage class has it, csi provisioner will pass it to the zfs plugin.

Reply:
I saw this. https://github.com/openebs/zfs-localpv/blob/d501cc8a996b3db00e5935979ea44ee5f206add8/pkg/zfs/mount.go#L18 

Thanks!